tests/tensor_search/integ_tests/test_search_unstructured.py

```diff
@@ -519,6 +536,51 @@ def test_filtering(self):
                 if expected_id:
                     self.assertEqual(expected_id, res["hits"][0]["_id"])
 
+    def test_filtering_fake_and_real_boolean_fields(self):
```

Comment:
I find the fake and real boolean terminology confusing. How about `test_filter_string_boolean_fields`?

Reply:
done